docs/extend/plugins.md

```diff
@@ -65,6 +65,12 @@ Plugin
 [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume)                   | A volume plugin that connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
+### Authorization plugins
+
+| Plugin                                                       | Description                                                                                                                                                                                           |
```

Comment:
For consistency with the other tables in this document, can you remove the leading and trailing `|` in the table?